### DISCUSSION TITLE
Group/Categories and Sort Tags

### DISCUSSION THREAD

**Discussion by pat-githubber** [TiddlyWiki/TiddlyWiki5#6290] [Q&A]
Hi,

I just saw this way of categorizing and sorting tags within a color-coded group (see image below of the _tag manager_).
How do I do that, meaning:

1. Grouping tags (with color or icon or another way)
2. Sorting tags within group alphabetically 

My nooby approach was to start the tag with a prefix so that I had them closely located together when selecting tags for a tiddler. For example, tags would look like this:

- Parameter: Costs
- Parameter: Lifetime
- Parameter: Capacity
- Tech: Wind
- Tech: Solar
- Tech: Biomass
- etc.

![image](https://user-images.githubusercontent.com/87087327/143783577-4d81cfe2-9ab5-4e55-b4f1-d1551301f2a2.png)

**Comment by kookma** [comment]
Try this:

```
<$list filter="[all[tiddlers]tags[]!is[system]] :sort::reverse[{!!color}]">
<div class="tc-menu-list-item" style="margin-bottom:4px">
<<tag>>
</div>
</$list>
```

The above code sorts by color! See also https://talk.tiddlywiki.org/t/sort-tags-by-color/1701
